### ISSUE TITLE
nuget_client.download_package() hangs

### ISSUE THREAD

**Issue by thisisnaty** [microsoft/azure-devops-python-api#185]
I'm trying to download a NuGet package from a feed. Getting the package from the feed works fine, but when I try to download it with the NuGet client, it just hangs.

```python
feed_client = connection.clients_v5_0.get_feed_client()
nuget_client = connection.clients_v5_0.get_nuget_client()

package = feed_client.get_package(feed_id, package_guid)
package_version = package.versions[0].version

nuget_client.download_package(feed_id, package_name, version)
```

Any ideas what could be happening?

**Comment by tedchamb** [comment]
Try adding this code before your call and rerunning, so we can get a better idea where it is hanging:

```
import logging

logging.basicConfig(level=logging.DEBUG)
```

**Comment by tedchamb** [comment]
I have a fix in #213

**Comment by tedchamb** [comment]
Fixed in release: [5.0.0b6](https://github.com/microsoft/azure-devops-python-api/releases/tag/5.0.0b6)
